.travis.yml

```diff
@@ -21,8 +32,11 @@ install:
 - export PATH=$PATH:/tmp/glide/${GLIDE_OS_ARCH}
 
 script:
-- travis_retry make test
+- make
```

Comment:
Can we create a `make travis` or `make ci` target instead of all these individual targets in here? Or add all targets one line (`make test cross fmtcheck` etc)